tests/kernel/ltp_setup_networking.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This module installs and setup services and other things needed for
+# networking part of the LTP (Linux Test Project).
+# Maintainer: Petr Vorel <pvorel@suse.cz>
+
+use 5.018;
+use warnings;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+
+sub install_services {
+    my $self = shift;
+
+    # utils
+    zypper_call("in expect iputils net-tools-deprecated telnet");
```

Comment:
It is better to call `zypper_call` with `, log => 1` when we are using the serial terminal because otherwise Zypper fills up pages of the log with terminal codes and dots. 

I am waiting for more people to start using the serial terminal before coming up with a more general fix for that.